grafana/grafana
# Loki: remove angular

remove all angular code from the Loki datasource.

how to test:

1. have a running loki datasource, for example `make devenv sources=loki`
2. run main-branch grafana (basically any grafana that is not this pull-request), create a loki-annotation, observe how the stored annotation looks like in the JSON (go to the dashboard-settings, click on JSON_model)
3. now switch to this PR, open the annotation-query-editor, and check that it loaded correctly. modify things, save them, check the JSON that it got updated correctly
4. check on the dashboard visually that the changes got applied correctly.
